pandas/plotting/_core.py

```diff
@@ -608,6 +608,7 @@ class PlotAccessor(PandasObject):
         - 'hexbin' : hexbin plot.
 
     figsize : a tuple (width, height) in inches
+        The size of the figure to create in matplotlib.
```

Comment:
Ideally we want to only have the type in the line with the name. `figsize : tuple of (float, float)` for example.

Then, in the description you can explain it's width and height, and that the units are inches.

Reply:
So i'll try to change the description to:
```
    figsize : a tuple (tuple, tuple)
        The size (width, height) of the figure in inches to create in matplotlib.
```
Is it enough?

Reply:
Much better, but the type is not a tuple of tuples I think it's it. Also, just say "tuple...", not "a tuple...".

I would also not say that is the matplotlib figure, just the figure, since we can plot with different libraries.

Reply:
Oh yea, I forgot about it. Gonna fix it right now.